### PR TITLE
Add backends and ensemble option to L0_infer

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: qa/L0_infer/infer_test.py [hunk 1]
```diff
@@ -39,6 +39,11 @@
 TEST_SYSTEM_SHARED_MEMORY = bool(int(os.environ.get('TEST_SYSTEM_SHARED_MEMORY', 0)))
 TEST_CUDA_SHARED_MEMORY = bool(int(os.environ.get('TEST_CUDA_SHARED_MEMORY', 0)))
 CPU_ONLY = (os.environ.get('TENSORRT_SERVER_CPU_ONLY') is not None)
+BACKENDS = os.environ.get('BACKENDS', "graphdef savedmodel netdef onnx libtorch plan custom")
+ENSEMBLES = bool(int(os.environ.get('ENSEMBLES', 1)))
+
+if ENSEMBLES:
+    BACKENDS="graphdef savedmodel netdef onnx libtorch plan custom"
```
Comment (GuanLuo) [inline qa/L0_infer/infer_test.py]: Don't override `BACKENDS`, in that way we have finer control over "ensemble" x "backend" combinations.

Comment (CoderHam) [reply qa/L0_infer/infer_test.py]: Done

File: qa/L0_infer/test.sh [hunk 2]
```diff
@@ -62,6 +62,16 @@ if [ "$TENSORRT_SERVER_CPU_ONLY" == "1" ]; then
     fi
 fi
 
+# If BACKENDS not specified, set to all
+if [ -z "$BACKENDS" ]; then
```
Comment (deadeyegoodwin) [inline qa/L0_infer/test.sh]: Can you use bash syntax like this?

BACKENDS=${BACKENDS:="graphdef savedmodel netdef onnx libtorch plan custom"}

File: qa/L0_infer/test.sh [hunk 3]
```diff
@@ -62,6 +62,16 @@ if [ "$TENSORRT_SERVER_CPU_ONLY" == "1" ]; then
...
+    BACKENDS="graphdef savedmodel netdef onnx libtorch plan custom"
+fi
+
+# If ENSEMBLES not specified, set to 1
+if [ -z "$ENSEMBLES" ]; then
+    ENSEMBLES="1"
```
Comment (deadeyegoodwin) [inline qa/L0_infer/test.sh]: Use above syntax

File: qa/L0_infer/test.sh (hunk 2 above)
Comment (CoderHam) [reply qa/L0_infer/test.sh]: Done

File: qa/L0_infer/test.sh (hunk 3 above)
Comment (CoderHam) [reply qa/L0_infer/test.sh]: Done

**Pull request merged**